main/settings.py

```diff
@@ -49,6 +49,8 @@
     # Celery
     CELERY_BROKER_URL=str,
     CELERY_RESULT_BACKEND=str,
+    # Cache
```

Comment:
We also need to add a setting for TEST to make sure it doesn’t affect the local development cache server.

This should be optional and only required when running test (not required in helm)

Example:

**settings.py**
https://github.com/mapswipe/mapswipe-backend/blob/develop/main/settings.py#L99
https://github.com/mapswipe/mapswipe-backend/blob/develop/main/settings.py#L413

**Test base class**
https://github.com/mapswipe/mapswipe-backend/blob/develop/main/tests/base_test.py#L16-L28
https://github.com/mapswipe/mapswipe-backend/blob/develop/main/tests/base_test.py#L48